auras.py

```diff
@@ -191,54 +236,67 @@ def scaled_value(self, value: int, aura_effect: int, index_handlers: list[str])
 			value = int(value)
 		return value
 
-	def ascendancy_mod(self, num_auras: int, node_stats: list[str]) -> list[str]:
+	def ascendancy_mod(self, aura_counter: list[int], node_name: str) -> list[str]:
 		# TODO: champion
-		# TODO: scale mods by character aura effect
-		if len(node_stats) == 5 and node_stats[0] == 'Melee Hits have 50% chance to Fortify':
+		if node_name == "Champion":
 			return [
 				'// champion',
 				'Enemies Taunted by you take 10% increased Damage',
 				'Your Hits permanently Intimidate Enemies that are on Full Life',
 			]
-		elif len(node_stats) == 5 and node_stats[0] == '25% reduced Effect of Curses on you':
+		elif node_name == "Guardian":
 			return [
 				'// guardian',
-				f'+{num_auras}% Physical Damage Reduction',
+				f'+{sum(int((1 + effect/100) * 1) for effect in aura_counter)}% Physical Damage Reduction',
```

Comment:
wait is that how that works? each individual aura grants the 1% PDR so they're individually affected by aura effect? so at 99% increased aura effect, you gain no bonus, but at 100%, you double the additional PDR?

Reply:
yup. thats why hitting the thresholds is so important